tests/baselines/reference/quickInfoThrowsTag.baseline

```diff
@@ -0,0 +1,179 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoThrowsTag.ts",
+      "position": 170,
+      "name": "1"
+    },
+    "quickInfo": {
+      "kind": "function",
+      "kindModifiers": "",
+      "textSpan": {
+        "start": 168,
+        "length": 2
+      },
+      "displayParts": [
+        {
+          "text": "function",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "f1",
+          "kind": "functionName"
+        },
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [],
+      "tags": [
+        {
+          "name": "throws"
+        }
+      ]
+    }
+  },
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoThrowsTag.ts",
+      "position": 175,
+      "name": "2"
+    },
+    "quickInfo": {
+      "kind": "function",
+      "kindModifiers": "",
+      "textSpan": {
+        "start": 173,
+        "length": 2
+      },
+      "displayParts": [
+        {
+          "text": "function",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "f2",
+          "kind": "functionName"
+        },
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "documentation": [],
+      "tags": [
+        {
+          "name": "throws",
+          "text": [
```

Comment:
shouldn't this include 'Error' somewhere? Maybe I'm misunderstanding the baseline json format, but I expected that the tag info should include the type name somewhere.

Reply:
I've added changes to include information about the type. @sandersn I noticed that the `@returns` tag also doesn't include this info. Is it ok? Or do we need to create a separate issue and fit it?
 
```
////class E extends Error {}
/////**
//// * @returns {E} comment
//// */
////function f1() {}
////f1/**/()

   "documentation": [],
   "tags":[
      {
         "name":"returns",
         "text":[
            {
               "text":"comment",
               "kind":"text"
            }
         ]
      }
   ]

```